src/main/java/com/microsoft/sqlserver/jdbc/Util.java

```diff
@@ -373,9 +373,14 @@ else if (ch == ':')
                         name = SQLServerDriver.getNormalizedPropertyName(name, logger);
                         if (null != name) {
                             if (logger.isLoggable(Level.FINE)) {
-                                if ((false == name.equals(SQLServerDriverStringProperty.USER.toString()))
-                                        && (false == name.equals(SQLServerDriverStringProperty.PASSWORD.toString())))
-                                    logger.fine("Property:" + name + " Value:" + value);
+                                if (false == name.equals(SQLServerDriverStringProperty.USER.toString())) {
+                                    if (!name.contains("password") && !name.contains("Password")) {
```

Comment:
either works.
only reason i wrote it this way is that a similar if condition is used for  SQLServerDataSource

```
private void setStringProperty(Properties props,
            String propKey,
            String propValue) {
        if (loggerExternal.isLoggable(java.util.logging.Level.FINER) && !propKey.contains("password") && !propKey.contains("Password")) {
            loggerExternal.entering(getClassNameLogging(), "set" + propKey, propValue);
        }
        else
            loggerExternal.entering(getClassNameLogging(), "set" + propKey);
        if (null != propValue)
            props.setProperty(propKey, propValue);
        loggerExternal.exiting(getClassNameLogging(), "set" + propKey);
    }
```